test/evergreen.yml

```diff
@@ -79,6 +79,49 @@ functions:
               export PATH=/opt/mongodbtoolchain/v4/bin:$PATH
             fi
 
+  "get engflow creds": &get_engflow_creds
+  - command: s3.get
+    display_name: "get engflow key"
+    params:
+      aws_key: ${engflow_key}
+      aws_secret: ${engflow_secret}
+      remote_file: engflow/engflow.key
+      bucket: serverengflow
+      local_file: "mongo/engflow.key"
+  - command: s3.get
+    display_name: "get engflow cert"
+    params:
+      aws_key: ${engflow_key}
+      aws_secret: ${engflow_secret}
+      remote_file: engflow/engflow.cert
+      bucket: serverengflow
+      local_file: "mongo/engflow.cert"
+  - command: shell.exec
+    params:
+      display_name: "generate evergreen engflow bazelrc"
+      shell: bash
+      working_dir: mongo
+      script: |
+        set -o errexit
+        set -o verbose
+
+        # Pulled from evergreen/generate_evergreen_engflow_bazelrc.sh consider
+        # consolidating once prelude.sh is runnable in the perf project.
```

Comment:
Is there a public ticket we can link to for the `prelude.sh` work? If not I'm happy to leave this comment as is.

Reply:
I don't think we had one before but I created https://jira.mongodb.org/browse/SERVER-86966 to track it